code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -222,6 +226,9 @@
 			feedback_inc("cyborg_medical",1)
 
 		if("Security")
+			if(!config.allow_secborg)
+				src << "spam class = 'warning'>ERROR: NANOTRASEN HAS DISABLED THIS MODULE. PLEASE SELECT A DIFFERENT ONE!</span>"

```

Comment:
RemieRichards I actually coded that first before removing it from the list. I'll remove it.